app/mailboxes/application_mailbox.rb

```diff
@@ -5,5 +5,18 @@
 # # https://rubular.com/ can help you with your regex
 
 class ApplicationMailbox < ActionMailbox::Base
+  before_processing :ensure_user_present
```

Comment:
WDYTA `:ensure_user_presence` instead of `:ensure_user_present`? 'Present' sounds more like a gift. Another alternative could be ':ensure_user_is_present'